cupy/cuda/compiler.py

```diff
@@ -67,6 +71,36 @@ def _get_arch():
                _nvrtc_max_compute_capability)
 
 
+def _check_cudadevrt_needed(options):
```

Comment:
`check` sounds as if it raise an error depending on arguments.
```suggestion
def _is_cudadevrt_needed(options):
```